cpp/src_prims/distance/distance_fragment_multiply_add.h

```diff
@@ -18,7 +18,6 @@
 
 #include <cutlass/fragment.h>
 #include <cutlass/shape.h>
-#include "cuda_utils.h"
```

Comment:
Even though this file doesn't include any `.cuh` files, this should be renamed to a `.cuh` since it is including cutlass related device code.

Reply:
never mind. THis has been resolved in the latest commit.